spec/foxsi2_count_spectrum.pro

```diff
@@ -65,19 +68,39 @@ area = get_foxsi_effarea( $
 		energy=emid, module_num = module_num, use_theoretical=use_theoretical, $
 		offaxis_angle=offaxis, _extra=_extra )
 
-if keyword_set(stop) then stop
+;if keyword_set(stop) then stop
 
 ; fold flux with FOXSI response
+eff_fudge = (1-0.05)
 counts = flux*area.eff_area_cm2  ; now the units are counts per second per keV
 
 ; Smear to account for imperfect energy resolution (~0.5 keV).
 if keyword_set(smear) then begin
 	i=where( emid gt 3. and emid lt 30. )
 	nbins = 0.5 / average(ewid)
```

Comment:
Can we add a keyword for the energy resolution? Then this function could be used for CdTe also.